doc/releases/release-notes-3.3.rst

```diff
@@ -330,6 +330,22 @@ Kernel
 Architectures
 *************
 
+* ARC
+
+  * Fix & rework interrupt management (enabling / disabling) for the SMP systems
```

Comment:
Need to use the past tense; but, we will almost certainly not have time for another round of review. I will fix these after this PR is merged.